packages/block-library/src/search/edit.js

```diff
@@ -257,6 +257,7 @@ export default function SearchEdit( {
 						type="button"
 						className={ buttonClasses }
 						style={ buttonStyles }
+						aria-label={ __( 'Search icon' ) }
```

Comment:
Can you change this to just "Search"? I am not sure screen reader users really need the information that the button is a Search icon at this point. It also doesn't add any description as to what the icon is.

@talldan may feel differently about my suggestion.

Reply:
Agree, just 'Search' would be good.

Though I think it should be the same text as the other PR (https://github.com/WordPress/gutenberg/pull/38136). I've just noticed that uses the `label` attribute, which is something the user has no way to enter when the icon button is used. The PHP code should probably fall back to 'Search' text there too.

Reply:
@talldan I believe the PHP has the label set to "Search" by default.

Reply:
Yep, you're right! No change needed there then. 

If possible, it'd be great to have the code here work the same way.